app/lib/actions/katello/repository/clone_to_environment.rb

```diff
@@ -8,6 +8,8 @@ def plan(repository, environment, options = {})
           clone = find_or_build_environment_clone(repository, environment)
 
           sequence do
+            plan_action(Katello::Repository::CorrectChecksum, clone)
```

Comment:
I think we want to completely remove this action (even on sync), what was the reasoning for adding it here?

Reply:
> and instead call this function when creating or updating the 'cloned' repos on cv publish/promote.

I'm trying to do this part of your instructions, and may easily be confused ;)